zingolib/src/lightclient.rs

```diff
@@ -354,6 +363,26 @@ pub mod instantiation {
                 save_buffer: ZingoSaveBuffer::new(buffer),
             })
         }
+        /// This is the fundamental invocation of a LightClient. It lives in an asyncronous runtime.
+        #[cfg(feature = "sync")]
+        pub async fn create_from_wallet_async(wallet: LightWallet) -> io::Result<Self> {
+            let mut buffer: Vec<u8> = vec![];
+            wallet.write(&mut buffer).await?;
+            let config = wallet.transaction_context.config.clone();
+            Ok(LightClient {
+                wallet: Arc::new(Mutex::new(wallet)),
+                config: config.clone(),
+                #[cfg(not(feature = "sync"))]
```

Comment:
This cfg line can't be hit, as we're inside of a `cfg(feature = sync)` block

Reply:
good spot! fixing now